include/session.php

```diff
@@ -40,12 +40,12 @@ function ref_session_write ($id,$data) {
 	if($session_exists) {
 		$r = q("UPDATE `session`
 				SET `data` = '%s'
-				WHERE `data` != '%s' AND `sid` = '%s'",
+				WHERE `sid` = '%s' AND `data` != '%s'",
 				dbesc($data), dbesc($data), dbesc($id));
 
 		$r = q("UPDATE `session`
 				SET `expire` = '%s'
-				WHERE `expire` != '%s' AND `sid` = '%s'",
+				WHERE `sid` = '%s' AND `expire` != '%s'",
 				dbesc($expire), dbesc($expire), dbesc($id));
```

Comment:
Same here:
Shouldn't it rather be `dbesc($expire), dbesc($id), dbesc($expire));` because you have changed the order of `sid` and `expire` in the line above?